gordo/machine/model/anomaly/diff.py

```diff
@@ -231,24 +283,41 @@ def _scaled_mse_per_timestep(
     ) -> pd.Series:
         """
         Calculate the scaled MSE per timestep/sample
-
         Parameters
         ----------
         model: BaseEstimator
             Instance of a fitted :class:`~DiffBasedAnomalyDetector`
         y_true: Union[numpy.ndarray, pd.DataFrame]
         y_pred: Union[numpy.ndarray, pd.DataFrame]
-
         Returns
         -------
         panadas.Series
             The MSE calculated from the scaled y and y predicted.
         """
-        scaled_y_true = model.scaler.transform(y_true)
+        try:
+            scaled_y_true = model.scaler.transform(y_true)
+        except (NotFittedError, ValueError):
+            scaled_y_true = model.scaler.fit_transform(y_true)
         scaled_y_pred = model.scaler.transform(y_pred)
         mse_per_time_step = ((scaled_y_pred - scaled_y_true) ** 2).mean(axis=1)
         return pd.Series(mse_per_time_step)
 
+    @staticmethod
+    def _absolute_error(
+        y_true: Union[pd.DataFrame, np.ndarray],
+        y_pred: Union[pd.DataFrame, np.ndarray],
+    ) -> pd.DataFrame:
+
+        return pd.DataFrame(np.abs(y_true - y_pred))
```

Comment:
I guess this comes down to personal preference, I find it more elegant having that method, since it is used more than once in the code and with the method name, one knows immediately what it is doing.